audb/core/load.py

```diff
@@ -1156,38 +1270,50 @@ def load_table(
     cached_versions = None
 
     db_root = database_cache_folder(name, version, cache_root)
+    db_lock_path = database_lock_path(db_root)
+    result = None
```

Comment:
Maybe `df` instead of `result`?

Reply:
Done. I also renamed `result` to `files' in `load_media()`.